.github/issue_template.md

```diff
@@ -1 +1,28 @@
-issue
+{{Description of feature/bugfix here}}
+
+[PFXXX-XXXX] \<Issue Title\>
+ 
+I want to **\<request feature/report bug\>**
+
+- **Current behavior**
+
+   What is the current behavior?
+
+   > If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem
+
+- **Expected behavior**
+
+   What is the expected behavior?
+   
+- **Motivation/Use case for changing behavior**
+
+   What is the motivation / use case for changing the behavior?
+
+- **Please tell us about your environment**:
+
```

Comment:
change format to:

issue type : [ in git convention]

Priority: <low med high>

User Story:
<link to user story ( can be either trello or figma)>

Expected date:
start:
end:

Notes: